.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,10 +36,13 @@ relevant to every Crossplane pull request.
 I have:
 - [ ] Run `make reviewable` to ensure this PR is ready for review.
 - [ ] Ensured this PR contains a neat, self documenting set of commits.
-- [ ] Updated any relevant [documentation], [examples], or [release notes].
-- [ ] Updated the RBAC permissions in [`clusterrole.yaml`] to include any new types.
+- [ ] Updated any relevant [documentation] and [examples].
```

Comment:
I notice this removes 'release notes' from the checklist. I think I'm on board with this because I don't think I've ever seen a Crossplane PR actually update the release notes.

Reply:
i actually dislike that, it makes the burden of authoring the release notes greater when it comes time for release.  I would actually like to see the pending release notes get utilized and be a check item that is respected in this PR template.  We've had success with this in Rook, e.g. the release notes basically take shape **during** the milestone: https://github.com/rook/rook/blob/master/PendingReleaseNotes.md

Reply:
i don't think it's a particularly strong argument to say we should not include the pending release notes in the definition of done just because they aren't currently being used.  i think this is an opportunity to define what our process is going forward, we can establish and make changes to the process right now.  That being said, I won't hold this PR up for it.